Profiler/CProfilerCallback.cpp

```diff
@@ -36,9 +36,9 @@ namespace {
 	/** The version of the profiler */
 	const char* PROFILER_VERSION_INFO =
 #ifdef _WIN64
-		"Coverage profiler version 0.11.0 (64bit)"
+		"Coverage profiler version 0.11.1 (64bit)"

```

Comment:
Actually I'd like to switch to versioning with YY.MM.xx like Christian proposed, e.g. 18.2.0. WOuld also be cool to get the version injected from the build process. What do you think?

I'll have a look at injecting later...

Reply:
See https://github.com/cqse/teamscale-profiler-dotnet/pull/9 :)

Reply:
Done :)